### DISCUSSION TITLE
404

### DISCUSSION THREAD

**Discussion by gojo22k** [vercel/community#6170] [Help]
### Summary

i am facing an error when using the serverless functionality of yous in locally working but not in the deployment 

here is my code please check index.js 
const express = require('express');
const axios = require('axios');

const app = express();

// Define the endpoint for fetching FileMoon data
app.get('/filemoon', async (req, res) => {
    try {
        // Extract necessary parameters from the request
        const { apiKey, folderId } = req.query;

        // Make HTTP request to FileMoon API to fetch folder list
        const fileMoonApiUrl = `https://filemoonapi.com/api/folder/list?key=4264362356tvlyudblyuabfulb&fld_id=0`;
        const response = await axios.get(fileMoonApiUrl);

        // Return the data back as-is
        res.json(response.data);
    } catch (error) {
        console.error('Error fetching folder list from FileMoon API:', error);
        res.status(500).json({ error: 'Internal Server Error' });
    }
});

module.exports = app;

### Steps to Reproduce

solutions please

**Comment by amyegan** [comment]
Hey @gojo22k. If you include a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) or share your repo, someone here may be able to give specific suggestions.

I also recommend taking a look at our guide about [using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel). The setup is a little different with Serverless Functions than it would be on a traditional server. I built a minimal example based on an older version of that guide in case it helps to see a full repo: https://github.com/amyegan/express-vercel

**Reply by gojo22k** [reply]
thank for assistance and replying my silly question 
but i solved the by changing from express.js to next.js

**Comment by gojo22k** [comment]
thank for assistance and replying my silly question
but i solved the by changing from express.js to next.js
